jbdevstudio/com.jboss.jbds.central.discovery/plugin.xml

```diff
@@ -59,7 +59,7 @@ reasonable, reporting issues to these providers as required.</description>
             categoryId="org.jboss.tools.central.discovery.a.web"
             groupId="org.jboss.tools.central.discovery.a.web.core"
             certificationId="com.jboss.jbds.discovery.certification.supported"
-            description="AngularJS Tooling support for Ionic framework"
+            description="AngularJS Tooling provides content assist and navigation + Ionic framework support"
```

Comment:
Thanks, I must have copied & pasted from the wrong plugin.xml -- they all looked the same until you pointed out that having tern in the EA version mean the addition of "navigation" in the description. Fixed and force pushed. 

Better?